README.adoc

```diff
@@ -77,10 +72,10 @@ Maven goals to build the application and run it inside Open Liberty:
 mvn install liberty:start-server
 ```
 
-Point your browser to http://localhost:9080/inventory/systems. This is the staring point of the `inventory`
+Point your browser to the http://localhost:9080/inventory/systems URL. This is the staring point of the `inventory`
 service and it displays the current contents of the inventory. As you might expect, these are empty since
```

Comment:
Best practice to name the thing:
the .... URL

Reply:
We refer to URLs throughout the guide. I think they're indicative as themselves, don't need 'the .... URL'. In this guide's context, the URLs are the service endpoints.